src/test/v3-dns-did-signed.spec.ts

```diff
@@ -1,8 +1,9 @@
-import { uploadDocument, validateIframeTexts, validateIssuerTexts, location } from "./helper";
+import { uploadDocument, validateIframeTexts, validateIssuerTexts, location, navigateToVerify } from "./helper";
 
 fixture("v3 DNS DID Certificate Rendering").page`${location}`;
 
 test("sample document is rendered correctly when dns did is verified", async () => {
+  await navigateToVerify();
```

Comment:
w8 then how did this integration test work when i was working on it previously?
did logic for `uploadDocument` recently change?

Reply:
`(did dns fixture wasn't included in glob pattern for tests)`